### ISSUE TITLE
No report is written if a benchmark run crashes before finishing

### ISSUE THREAD

**Issue by gnuvince** [Sable/wu-wei-benchmarking-toolkit#28]
If benchmarks 1, 2, and 3 have been correctly executed, but benchmark 4 fails, `wu report` will not show the timings for benchmarks 1, 2, 3.  This is annoying when running a large number of long-running benchmarks and one fails; the entire suite needs to be re-executed rather than resuming from the failure point.

**Comment by elavoie** [comment]
Yes. The work around for now is to first execute all the benchmarks once with a small input to check they all execute correctly, and then launch the long running ones for peformance analysis.
